seed/csharp-sdk/imdb/no-custom-config/src/SeedApi.Test/Utils/JsonDiffChecker.cs

```diff
@@ -0,0 +1,81 @@
+namespace SeedApi.Test.Utils;
+
+using System.Collections.Generic;
+using Newtonsoft.Json.Linq;
+
+using System.Collections.Generic;
+using Newtonsoft.Json.Linq;
+using NUnit.Framework;
+
+public static class JsonDiffChecker
+{
+    public static void AssertJsonEquals(string jsonString1, string jsonString2)
```

Comment:
chatgpt helped with this. will need to test more, but we saw how hard the jtoken `true` or `false` comparison was to debug, and this helps a lot